### PR TITLE
tests/integration: use asyncLoadScript()

### PR DESCRIPTION
**Pull request by alxndrsn**
Replace duplicated script-loading code with existing `commonUtils.asyncLoadScript()` function.

Also removes a timeout-based implementation in `tests/integration/test.attachments.js`.

### CODE REVIEW

File: tests/integration/browser.migration.js [hunk 1]
```diff
@@ -46,13 +46,7 @@ describe('migration', function () {
       if (!match) {
         return testUtils.Promise.resolve();
       }
-      return new testUtils.Promise(function (resolve, reject) {
-        var script = document.createElement('script');
-        script.onload = resolve;
-        script.onerror = reject;
```
Comment (lucidNTR) [inline tests/integration/browser.migration.js:52]: the asyncLoadScript does not have the error handler, would you mind adding it there with the promise rejection as here?

Comment (lucidNTR) [reply tests/integration/browser.migration.js:52]: in addition, i think asyncLoadScript will fail if there is no existing script tag due to prepending document.getElementsByTagName('script')[0], if we replace existing code we might also add the fallback behaviour there to fall back to append child to body if there is no existing script tag.

File: tests/integration/test.attachments.js [hunk 2]
```diff
@@ -3113,15 +3113,9 @@ adapters.forEach(function (adapter) {
             var bigimage = require('./deps/bigimage.js');
             cb(null, bigimage);
           } else { // browser
-            var script = document.createElement('script');
-            script.src = 'deps/bigimage.js';
-            document.body.appendChild(script);
-            var timeout = setInterval(function () {
```
Comment (lucidNTR) [inline tests/integration/test.attachments.js:3119]: great this code will be gone

**Review by lucidNTR — CHANGES_REQUESTED** [review]

File: tests/integration/browser.migration.js (hunk 1 above)
Comment (alxndrsn) [reply tests/integration/browser.migration.js:52]: Good catch - thanks!  I'll update.

Comment (alxndrsn) [reply tests/integration/browser.migration.js:49]: > we might also add the fallback behaviour there to fall back to append child to body if there is no existing script tag

I've updated to just append all scripts to `document.body`.  It seems to work 🫣

File: tests/common-utils.js [hunk 3]
```diff
@@ -141,9 +141,7 @@ commonUtils.asyncLoadScript = function (url) {
       }
     };
 
-    // Attach the script tag to the page (before the first script) so the
-    //magic can happen.
-    firstScript.parentNode.insertBefore(script, firstScript);
+    document.body.append(script);
```
Comment (lucidNTR) [inline tests/common-utils.js:144]: good catch, for script async = true the order does not matter as per spec, and even if , the order would mostly depend on call order of asyncLoadScript, not tag order...

**Review by lucidNTR — APPROVED** [review]

**Pull request merged**